sockeye/decoder.py

```diff
@@ -452,6 +452,7 @@ class RecurrentDecoderConfig(Config):
     :param context_gating: Whether to use context gating.
     :param layer_normalization: Apply layer normalization.
     :param attention_in_upper_layers: Pass the attention value to all layers in the decoder.
+    :param e2d_lhuc: Apply LHUC for encoder to decoder initialization.
```

Comment:
`e2d_lhuc` seems a bit cryptic. Would it make sense to rename to `state_init_lhuc` and move it after `state_init`?